force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +308,11 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
+        Book hawking = new Book('A Brief History of Time', 'Stephen Hawking');
         // Call the getBookDetails method on the book object to get the details of the book
-
+        String bookDeets = hawking.getBookDetails();
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return bookDeets; // Replace null with the variable you used to store the result
```

Comment:
Good job. See below for a way to be more concise and I don't think you loose readability. 

 return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();